web/forms/implementing_simple_autocompletion/match_element.dart

```diff
@@ -17,7 +17,7 @@ class MatchElement extends PolymerElement {
   MatchElement.created() : super.created();
 
   @Observe('value, inputQuery')
-  valueChanged(_, __) => Polymer.dom(root).innerHtml = highlightedValue;
```

Comment:
Actually I would just deprecate it in favor of `new PolymerXxx()`

I don't know a use case where I would want the method accept either `Event` or `Element` because I don't know or don't care what type the value is that I actually hold, or even less where I don't expect a specific return type (`PolymerDom` or `PolymerEvent`) 

I also don't think that it is too important to have the same method as in JS.
Just mention it in the docs.